pandas/core/arrays/datetimelike.py

```diff
@@ -468,6 +542,67 @@ def _values_for_factorize(self):
     def _from_factorized(cls, values, original):
         return cls(values, dtype=original.dtype)
 
+    def _values_for_argsort(self):
```

Comment:
not a big  deal, but did you intend to include values_for_argsort? Maybe update the title?

Though I suspect this isn't being tested yet.

Reply:
I think past-me included this in the hopes of making rebasing \epsilon simpler.